src/components/MovieReviewItem/index.js

```diff
@@ -25,20 +31,43 @@ const MovieReviewItem = ({
             {/*        <MovieItem movie={review.movie} posterOnClickHandler={() => {}}/>*/}
             {/*    </div>*/}
             {/*}*/}
-            <div className="col-12 card border-0 bg-light">
-                <div className="card-body d-flex justify-content-between">
-                    <div>
-                        <h5 className="card-title p-1">User: {review.postedBy.username}</h5>
-                        <h6 className="card-subtitle text-muted p-1 ">Rating: {review.rating}</h6>
-                        <p className="card-text p-1">{review.review}</p>
-                    </div>
-                    <div>
+            <div className="col-12 border-0 bg-light">
+                <div className="">
+                {location.pathname === '/profile/s/reviews' && 
+                <div className="row p-4" onClick={() => onClickNavigate(review.movie)}>
```

Comment:
This on click handler should be put with the image, or the div wrapping the image, not with the whole big div

Reply:
Done. I have put it on the image.